src/components/shared/TrackContextMenu.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'm not so sure about the location and the name of this file. We have a `TrackContextMenu.js` inside the`src/components/timeline/` directory, so I wanted to put this inside the `src/components/shared/` with the same name. But since these files have the same name, it could be confusing. What do you think?

Reply:
Hm yeah, I think `ContextMenuNoHidingOnEnter` would be better. I don't also have a strong preference to either to make it a local inside the file or to create a new file. But I felt like it would be cleaner with the new one.